src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.js

```diff
@@ -126,6 +129,13 @@ class ApprovalCommentForm extends React.Component<Props, State> {
         this.setState({ commentEditorState: nextEditorState });
 
     onApprovalDateChangeHandler = (date: Date): void => {
+        // The date given to us is midnight of the date selected.
+        // Modify date to be the end of day (minus 1 millisecond) for the given due date
+        date.setHours(TWENTY_THREE);
```

Comment:
this can be a one liner https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours

Reply:
1000ms not 60, so 999 for millis. Or get next day in millis minus 1

Reply:
Another possible option would be passing in -1, `date.setHours(0,0,0,-1)`

Reply:
BRUI has addTime() in datetime.js